Lib/distutils/_msvccompiler.py

```diff
@@ -129,11 +112,7 @@ def _find_vcvarsall(plat_spec):
         log.debug("%s cannot be found", vcvarsall)
         return None, None
 
-    if not vcruntime or not os.path.isfile(vcruntime):
-        log.debug("%s cannot be found", vcruntime)
-        vcruntime = None
-
-    return vcvarsall, vcruntime
+    return vcvarsall, None
```

Comment:
Do we even need to still return a second value?

Reply:
Strictly no, but I'm worried about libraries having overridden it (against instruction, but not really against design). Less risky to leave it there.

Reply:
_Especially_ since I want to backport this. Keeping it to just a behaviour change is objectively less likely to break anyone than an interface change :)